apps/desktop-wallet/src/features/thumbnails/NFTThumbnail.tsx

```diff
@@ -38,71 +38,77 @@ interface NFTThumbnailProps {
   onClick?: () => void
 }
 
-const NFTThumbnail = memo(
-  ({
-    nftId,
-    size,
-    border,
-    hideIfError,
-    borderRadius,
-    autoPlay,
-    showPlayIconIfVideo,
-    playOnHover,
-    onClick
-  }: NFTThumbnailProps) => {
-    const { t } = useTranslation()
-    const theme = useTheme()
-
-    const { data: nft, error, isLoading } = useFetchNft({ id: nftId })
-
-    if (error && !hideIfError)
-      return (
-        <ErrorMessage>
-          <X />
-        </ErrorMessage>
-      )
-
-    const src = nft?.image
-
-    return (
-      <Container
-        style={{
-          borderRadius: borderRadius || 0,
-          boxShadow: border ? `0 0 0 1px ${theme.border.primary}` : 'initial',
-          height: size,
-          width: size
-        }}
-        onClick={onClick}
-      >
-        {!isLoading ? (
-          src ? (
-            nft.dataType === 'image' ? (
-              <Image src={src} height={size} width={size} />
-            ) : nft.dataType === 'video' ? (
-              autoPlay ? (
-                <video src={src} autoPlay={autoPlay} loop width="100%" height="100%" preload="auto" muted playsInline />
-              ) : (
-                <VideoThumbnail videoUrl={src} showPlayIcon={showPlayIconIfVideo} playOnHover={playOnHover} />
-              )
-            ) : (
-              <ErrorMessage>{t('Unsupported media type')}</ErrorMessage>
-            )
-          ) : (
-            <ErrorMessage>{t('No media')}</ErrorMessage>
-          )
-        ) : (
-          <SpinnerContainer>
-            <Spinner />
-          </SpinnerContainer>
-        )}
-      </Container>
+const NFTThumbnail = ({ border, borderRadius, onClick, hideIfError, ...props }: NFTThumbnailProps) => {
```

Comment:
I made many changes in this component cause some of the conditions we had before were not taken care of.